algorithms/glued_trees/glued_trees.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>We run CI tests on each of the notebooks. We try to avoid long runs of the notebooks (more than 2-3 minutes). so please leave  this line of code in a comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>